guava/src/test/java/com/fasterxml/jackson/datatype/guava/failing/RangeStringFormatTest.java

```diff
@@ -0,0 +1,72 @@
+package com.fasterxml.jackson.datatype.guava.failing;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.guava.ModuleTestBase;
+import com.fasterxml.jackson.datatype.guava.deser.util.RangeFactory;
+import com.google.common.collect.Range;
+
+import java.time.Duration;
+import java.time.LocalDate;
+
+public class RangeStringFormatTest extends ModuleTestBase
+{
+    private final ObjectMapper MAPPER = mapperWithModule();
+
+    static class Stringified<T extends Comparable<?>>
+    {
+
+        @JsonFormat(shape = JsonFormat.Shape.STRING)
+        public Range<T> r;
+
+        public Stringified() {
+        }
+
+        public Stringified(Range<T> r) {
+            this.r = r;
+        }
+
+        public Range<T> toRange() {
+            return r;
+        }
+    }
+
+    public void testRangeSerializationToString() throws Exception
```

Comment:
Can we at least separate (by method of your choice) failing ones to passing tests?

Reply:
⁉️

All of these would fail as of now. I am working on implementing the serialization for this.